man/systemd.network.xml

```diff
@@ -1753,6 +1753,35 @@
         </varlistentry>
       </variablelist>
```

Comment:
this commit should carry "Fixes: #4042" in the commit text so that github can automatically close the issue when this is merged.

Reply:
ok